loadtests/simulation.py

```diff
@@ -0,0 +1,318 @@
+import json
+import os
+import random
+import uuid
+
+from konfig import Config
+from loads.case import TestCase
+from requests.auth import HTTPBasicAuth
+
+
+ACTIONS_FREQUENCIES = [
```

Comment:
I just took it from kinto and readinglist. Would it be ok to improve it in another iteration?